backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -404,6 +408,7 @@ public Response deleteEndpoint(@Context SecurityContext sec, @PathParam("id") UU
     @PUT
     @Path("/{id}/enable")
     @Produces(TEXT_PLAIN)
+    @Operation(summary = "Enable an endpoint", description = "Enables an endpoint that is disabled so that the endpoint will be executed after an operation that uses the endpoint is restarted. An operation must be restarted to use the enabled endpoint.")
```

Comment:
We don't restart operations, they only run once.

```suggestion
    @Operation(summary = "Enable an endpoint", description = "Enables an endpoint that is disabled so that the endpoint will be executed on the following operations that uses the endpoint.")
```